index.html

```diff
@@ -1,17 +1,20 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
     <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <link href="style.css" rel="stylesheet" />
+        <title>Microverse Second Trial</title>
```

Comment:
Hi @ShahierNashaat 
in the link tag, you should start with the rel attribute then the href attribute